internal/rbd/encryption.go

```diff
@@ -54,6 +54,10 @@ const (
 
 	// image metadata key for encryption
 	encryptionMetaKey = ".rbd.csi.ceph.com/encrypted"
+
+	// metadataDEK is the key in the image metadata where the (encrypted)
+	// DEK is stored.
+	metadataDEK = ".rbd.csi.ceph.com/dek"
```

Comment:
once we mirror the image the image metadata should also get mirroring. I will check this one.

Reply:
I expect that image metadata is automatically mirrored. But definitely good to check.